webapp/channels/src/components/admin_console/__snapshots__/schema_admin_settings.test.tsx.snap

```diff
@@ -620,7 +620,11 @@ exports[`components/admin_console/SchemaAdminSettings should match snapshot with
               }
             }
             disabled={false}
-            helpText={<span />}
+            helpText={
+              <span>
+                
+              </span>
+            }
```

Comment:
This was updated since it's failing when running the test bit it's to be removed by a separate PR - https://github.com/mattermost/mattermost/pull/34472/files#diff-3820336f890836f83a04117f4ed599448866a1098f5b6fd3e71203eec8011ff8